src/reaper_osara.cpp

```diff
@@ -441,6 +441,10 @@ bool isItemSelected(MediaItem* item) {
 	return *(bool*)GetSetMediaItemInfo(item, "B_UISEL", NULL);
 }
 
+bool isFreeItemPositioningEnabled(MediaTrack* track) {
+	return *(int*)GetSetMediaTrackInfo(track, "B_FREEMODE", NULL);
```

Comment:
Ah crap, I missed this. That should be `bool*`, not `int*`, but it doesn't really matter. I should go to bed apparently.